src/main/java/com/blockout/Log.java

```diff
@@ -0,0 +1,29 @@
+package com.blockout;
+
+import com.minecolonies.lib.Constants;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Logging utility class
+ */
+public class Log
+{
+    /**
+     * Mod logger
+     */
+    private static Logger logger = null;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "logger" logger private static final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1312)